tests/test_fips.py

```diff
@@ -152,3 +186,62 @@ def fips_mode_setup_check(container_per_test: ContainerData) -> None:
 )
 def test_openssl_fips_hashes(container_per_test: ContainerData):
     openssl_fips_hashes_test_fnct(container_per_test)
+
+
+@pytest.mark.parametrize(
+    "container_per_test", FIPS_GCRYPT_TESTER_IMAGES, indirect=True
+)
+@pytest.mark.skipif(
```

Comment:
If this is a bug that should be fixed, then the `skipif` should become an `xfail`
```suggestion
@pytest.mark.xfail(
```

Reply:
fixed